src/pricecypher/config_sections.py

```diff
@@ -0,0 +1,52 @@
+from pricecypher.endpoints import ConfigEndpoint
+from pricecypher.models import ConfigSection
+from pricecypher.rest import RestClient
+
+
+class ConfigSections(object):
+    """
+    Config API. Exposes all available operations on dataset configs, like fetching the available sections, and
+    retrieving the key-value pairs of a section and parsing it to a dictionary.
+
+    :param str bearer_token: Bearer token for PriceCypher (logical) API. Needs 'read:configuration' scope.
+    :param str config_base: (optional) Base URL for PriceCypher config service API.
```

Comment:
Should dataset_id be included in this docstring?

Reply:
Yea it should, added in commit 365b657.